README.md

```diff
@@ -106,10 +135,56 @@ All commands are issued from the the root of the project folder.
  * (e.g. `<project_root>/platforms/ios/ThaliTest.xcodeproj`)
  * `~/Code/ThaliTest> open platforms/ios/ThaliTest.xcodeproj`
 
+### Unit Testing the Thali Cordova Plugin
+
+The Thali Cordova plugin uses the [Jasmine](http://jasmine.github.io/) Behavior Driven Development framework for JavaScript.
+
+#### Desktop Testing
+
+Testing is available on the desktop for the Thali Cordova Plugin which uses a mock object to simulate the native Cordova `Mobile` calls.
+
+To get started, install, Jasmine:
+```
+$ sudo [npm] install -g jasmine
+```
+
+Next, simply run Jasmine which then looks at the specifications in the `specs` folder.  
+```
+$ jasmine
+```
+
+#### Mobile Testing
+
+Testing is also available on the mobile devices as well.  This will use the regular `Mobile` calls directly through Cordova to talk to the underlying system.
+
+To get started, copy the files from `sample/android/www` or `sample/ios/www` to the `www` folder.
+
+Next, install the requirements for testing:
+```
+$ jx install
+```
+
+Finally, build the application and run it on your device.  The test results should be shown in your developer console.
```

Comment:
Have you ever actually run the tests on iOS? If only to see if they actually talk to the console as they all fail?